GameObject.java

```diff
@@ -45,6 +47,12 @@ public double getDirection()
 	{
 		return direction;
 	}
+	// the updates the need to happen the the teams should not mess with
```

Comment:
things like life going away over time will go in there. If teams edited it
they would be cheating.

On Fri, Feb 6, 2015 at 9:04 PM, automagicly <notifications@github.com>
wrote:

> In GameObject.java
> <https://github.com/UWFAI/Josh/pull/1#discussion_r24286217>:
>
> > @@ -45,6 +47,12 @@ public double getDirection()
> >  	{
> >  		return direction;
> >  	}
> > +	// the updates the need to happen the the teams should not mess with
>
> why exactly ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UWFAI/Josh/pull/1/files#r24286217>.
>